sklearn/preprocessing/_data.py

```diff
@@ -655,7 +655,7 @@ def fit(self, X, y=None):
 
         Parameters
         ----------
-        X : {array-like, sparse matrix}, shape [n_samples, n_features]
+        X : {array-like, sparse matrix}, shape (n_samples, n_features)
```

Comment:
we should change `, shape` to `of shape` at the same time. In addition, I think that the algorithms pointed out will work with dataframe as well.

```suggestion
        X : {array-like, sparse matrix, dataframe} of shape (n_samples, n_features)
```